@xen-orchestra/fs/src/smb-mount.js

```diff
@@ -0,0 +1,35 @@
+import { join } from 'path'
+import { parse } from 'xo-remote-parser'
+import { tmpdir } from 'os'
+
+import MountHandler from './_mount'
+
+export default class SmbMountHandler extends MountHandler {
+  constructor(
+    remote,
+    { mountsDir = join(tmpdir(), 'xo-smb-mounts'), ...opts } = {}
+  ) {
+    const { domain, host, password, username } = parse(remote.url)
+    super(remote, opts, {
+      type: 'cifs',
+      host: `\\\\${host}`,
```

Comment:
What about `share`?

Reply:
`share` doesn't exist with parse. I had a mistake. And I can't find `share` attribute in the `xo-remote-parser` code

Reply:
My bad, it appears the *share* is already in `host`.